### PR TITLE
Update 7.4 image

### PR DESCRIPTION
**Pull request by dperpeet**
Also don't upgrade any installed cockpit packages in our test images.

 * [x] image-refresh rhel-7-4
 * [x] fix `check-accounts` failure (dperpeet)
 * [x] fix `check-realms` failure (dperpeet)
 * [x] fix `check-setroubleshoot` failure (dperpeet)
 * [x] fix `check-shutdown-restart` failure (dperpeet)
 * [x] fix "image preparation failure" (pitti, fixed locally)
 * [x] fix `test-metrics` (pitti, fixed locally)
 * [x] fix `check-services TestServices.testCreateTimer` (pitti, fixed locally)
 * [x] fix `check-multi-machine` (pitti)
 * [x] fix `check-packages TestPackages.testBasic` (pitti, fixed locally)
 * [x] fix `check-docker-storage TestDockerStorageMapper.testDevmapper` (dperpeet)

### CODE REVIEW

File: bots/images/scripts/lib/fedora.install [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 set -ex
 
-skip="cockpit-doc-"
+# don't update already installed cockpit packages
+installed=$(rpm --query --all --queryformat "%{NAME}-\[0-9\]\n" "cockpit*")
+skip="cockpit-doc-[0-9]
+$installed"
+rpm -qa --queryformat "%{NAME} " "cockpit*"
```
Comment (martinpitt) [inline bots/images/scripts/lib/fedora.install]: This looks like a debugging leftover.

Comment (dperpeet) [inline bots/images/scripts/lib/fedora.install]: oops! good catch

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-7437-20170802-143535/

**Comment by cockpituous** [comment]
Task failed: http://fedorapeople.org/groups/cockpit/logs/image-refresh-7437-20170802-143535/

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-7437-20170802-151414/

**Comment by cockpituous** [comment]
image-refresh rhel-7-4 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-7-4-20170802-153939

File: test/verify/check-accounts [hunk 2]
```diff
@@ -139,6 +139,7 @@ class TestAccounts(MachineCase):
                 return value.strip()
         return None
 
+    @skipImage("No available with Cockpit 138", "rhel-7-4")
```
Comment (martinpitt) [inline test/verify/check-accounts]: *meep* critical grammar bug ☺

Comment (dperpeet) [inline test/verify/check-accounts]: are you saying it's "yes available"?

**Comment by dperpeet** [comment]
@martinpitt Your fixup commits look good, thanks!

**Comment by martinpitt** [comment]
[tests succeeded](https://fedorapeople.org/groups/cockpit/logs/pull-7437-20170803-145252-e02e6588-verify-rhel-7-4/log.html), so I force-pushed a cleanup rebase.

**Pull request merged**